cargo-dist/src/init.rs

```diff
@@ -799,6 +802,66 @@ fn get_new_dist_metadata(
         }
     }
 
+    // Special handling of the Homebrew installer
+    if meta
+        .installers
+        .as_deref()
+        .unwrap_or_default()
+        .contains(&InstallerStyle::Pkg)
+    {
+        let pkg_is_new = !orig_meta
+            .installers
+            .as_deref()
+            .unwrap_or_default()
+            .contains(&InstallerStyle::Pkg);
+
+        if pkg_is_new && orig_meta.mac_pkg_config.is_none() {
+            let prompt = r#"you've enabled a Mac .pkg installer. This requires a unique bundle ID;
+    please enter one now. This is in reverse-domain name format.
+    For more information, consult the Apple documentation:
+    https://developer.apple.com/library/archive/documentation/CoreFoundation/Conceptual/CFBundles/BundleTypes/BundleTypes.html#//apple_ref/doc/uid/10000123i-CH101-SW1"#;
+            let default = "".to_string();
+
+            let identifier: String = if args.yes {
+                default
+            } else {
+                let res = Input::with_theme(&theme)
+                    .with_prompt(prompt)
+                    .allow_empty(true)
+                    .interact_text()?;
+                eprintln!();
+                res
+            };
+            let identifier = identifier.trim();
+            if identifier.is_empty() {
+                return Err(DistError::MacPkgBundleIdentifierMissing {});
+            }
+
+            let prompt = r#"Please enter the installation prefix this .pkg should use."#;
```

Comment:
should probably specify the default, or we shouldn't even ask? Not sure how typical it is to overload this.

Reply:
Overloading it is pretty common. Many people are going to want the default `/usr/local`, which is in the `PATH` but shared with other software. Other people are going to prefer, say, `/opt/mypkg`, which is out of the default `PATH` but a prefix fully owned by them, etc.

I was going for an "accept NULL, unwrap to `/usr/local` later" approach. But I guess that means the default could shift under the user, which maybe isn't ideal - I guess "inline a default if not specified" is probably better.